app/models.py

```diff
@@ -471,3 +486,8 @@ def filter_null_value_fields(obj):
     return dict(
         filter(lambda x: x[1] is not None, obj.items())
     )
+
+
+def generate_new_service_id():
+    # TODO: Decide what we want G7 service IDs to look like and implement
```

Comment:
Do we expect to call the draft publishing endpoint before we fix the service id generation? If not,  we could make the method `raise NotImplementedError()`. Do uuid4 pass our validation rules for service ids?

I see the reason for doing this since we'll reset our database once we go live anyway, but it still feels slightly dangerous to roll out code that will knowingly create incorrect data.